sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -150,4 +150,21 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTDome</Subsystem>
+    <EFDB_Topic>MTDome_command_resetDrivesAz</EFDB_Topic>
+    <Description>Reset one or more AZ drives. This is necessary when exiting from FAULT state without going to Degraded Mode since the drives don't reset themselves.</Description>
+    <item>
+      <EFDB_Name>reset</EFDB_Name>
+      <Description>Desired reset action to execute on each AZ drive: 0 means don't reset, 1 means reset.</Description>
+      <IDL_Type>float</IDL_Type>
```

Comment:
I strongly suggest making the data type boolean instead of float. If you anticipate needing more values in the future then consider integer and an enum.

Reply:
Good idea. I'll do that.